components/molecules/HighlightCard/highlight-card.tsx

```diff
@@ -154,7 +160,17 @@ const HighlightCard: React.FC<HighlightCardProps> = ({
               {getIcon(icon)}
             </div>
             {/* Label: Text */}
-            <div className="text-sm text-slate-600   leading-none">{label ? label : "Label"}</div>
+            <div className={clsx("text-sm text-slate-600 leading-none", tooltip && "cursor-pointer")}>
```

Comment:
Not sure `cursor-pointer` is the type of hover cursor we want: it makes me think it's a link but clicking it doesn't do anything.

Reply:
Agree with @jpmcb here. No need for this cursor. When they hover, it will just appear.